pom.xml

```diff
@@ -1822,6 +1828,37 @@
             </build>
         </profile>
 
+        <profile>
+            <id>incremental</id>
+            <activation>
+                <property>
+                    <name>incremental</name>
+                </property>
+            </activation>
+            <properties>
+                <!-- the *local* main, not refs/remotes/... -->
+                <gib.referenceBranch>main</gib.referenceBranch>
+                <gib.disableSelectedProjectsHandling>true</gib.disableSelectedProjectsHandling>
+            </properties>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>io.github.gitflow-incremental-builder</groupId>
+                        <artifactId>gitflow-incremental-builder</artifactId>
+                        <extensions>true</extensions>
+                        <!-- https://github.com/gitflow-incremental-builder/gitflow-incremental-builder#configuration
+                             General recap: Anything that is directly set in <configuration> cannot be redefined via '-D...'!
+                             See also: https://github.com/gitflow-incremental-builder/gitflow-incremental-builder/issues/213 -->
+                        <configuration>
+                            <!-- pointless to attempt incremental build if something like mvnw was changed
+                                 (and also potentially wrong, given that independent-projects might not be built) -->
+                            <skipIfPathMatches>\.github[/\\].*|\.mvn[/\\].*|mvnw.*|.*\.adoc|.*\.md</skipIfPathMatches>
+                        </configuration>
```

Comment:
We might want to use https://github.com/gitflow-incremental-builder/gitflow-incremental-builder#gibexcludepathsmatching in order to skip rebuilds when changing READMEs etc.?

Feel free to open another issue about that though.